src/app/components/TradeDetailModal/style.scss

```diff
@@ -0,0 +1,7 @@
+@import '~Styles/variables';
+
+.arrowStyle {
```

Comment:
Avoid using redundant words like `style`, `class`, etc and use a name that describes what it's used for. Something like `changeArrow`